inc/document.class.php

```diff
@@ -457,23 +457,28 @@ function send() {
          die("Error file ".$file." does not exist");
       }
 
-      // don't download picture files, see them inline
-      $attachment = "";
-      $filename_parts = explode(".", $this->fields['filename']);
-      $extension = array_pop($filename_parts);
-      $extension = strtolower($extension);
-      if (!in_array($extension, array('jpg', 'png', 'gif', 'bmp'))) {
-         $attachment = " attachment;";
-      }
+      // Document is a web link #1677
+      if ($this->fields['filename'] == NULL && $this->fields['link'] !== NULL) {
+         header("Location: " . $this->fields['link']);
```

Comment:
Hum... Even if this fix should indeed solve the issue, we cannot using that way open the link in a new tab (`@target="_blank"`).
I guess we should keep the fix (for a last chance solution), but also handle differently the way the doc is displayed front side.